src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -164,4 +165,15 @@ public static void showVolunteerAtIndex(Model model, Index targetIndex) {
         assertEquals(1, model.getFilteredVolunteerList().size());
     }
 
+    /**
+     * Updates {@code model}'s filtered list to show only the event at the given {@code targetIndex} in the
+     * {@code model}'s address book.
+     */
+    public static void showEventAtIndex(Model model, Index targetIndex) {
+        assertTrue(targetIndex.getZeroBased() < model.getFilteredEventList().size());
+
+        Event event = model.getFilteredEventList().get(targetIndex.getZeroBased());
+        model.updateFilteredEventList(e -> e.equals(event));
```

Comment:
wow so smart? i did the same method but urs is faster

Reply:
nah Jordan helped me with this.